src/main.py

```diff
@@ -85,9 +104,9 @@ def run(options: Options) -> int:
         return 0
 
     if not options.function_indexes_or_names:
-        functions = list(all_functions.values())
+        funcs = list(all_functions.values())
```

Comment:
I think using `funcs` and `functions` to be two separate variables for different things is hard to read. Particularly later on when there's the `for ..., (func, function) in ...`.

I think `funcs` could be `function_names` (or `fn_names`, `func_names`, etc.)?